projects/packages/forms/src/blocks/contact-form/variations.js

```diff
@@ -69,12 +69,20 @@ const variations = [
 				],
 			],
 			[
-				'jetpack/button',
-				{
-					text: __( 'Contact Us', 'jetpack-forms' ),
-					element: 'button',
-					lock: { remove: true },
-				},
+				'core/buttons',
+				{},
+				[
+					[
+						'core/button',
+						{
+							text: __( 'Contact Us', 'jetpack-forms' ),
```

Comment:
```suggestion
							text: __( 'Contact us', 'jetpack-forms' ),
```

Good time to switch to sentence case while we're here.